### DISCUSSION TITLE
[Help Needed] How can I make Bar Graph with Labels as shown ?

### DISCUSSION THREAD

**Discussion by kamalpandey** [FormidableLabs/victory#2117] [Q&A]
I am integrating the victory chart in react native and I need such chart. Is there any way I can do this in react native?

<img width="425" alt="Screen Shot 2021-11-19 at 23 36 06" src="https://user-images.githubusercontent.com/4987931/142670194-be85fe31-8db9-4a1d-afd7-6b48ce81f805.png">

**Comment by becca-bailey** [comment]
Hi @kamalpandey, I'm going to convert this issue to a discussion thread to prevent it from getting lost in issue reports. Thanks!
